packages/nimble-components/src/rich-text/viewer/index.ts

```diff
@@ -27,6 +32,24 @@ export class RichTextViewer extends FoundationElement {
      */
     public viewer!: HTMLDivElement;
 
+    /**
+     * @internal
+     */
+    public mentions: RichTextMention[] = [];
+
+    /**
+     * @internal
+     */
+    public mentionsMap: Map<string, MentionInternals<RichTextMentionConfig>> = new Map();
```

Comment:
What do you think about giving these names that distinguish the type of mention object in addition / instead of the data structure that stores the mention object? Maybe `mentionElements` and `mentionConfigs`/`mentionIdToConfigMap`/`mentionIdToMentionInternalsMap`? 

Just a half-formed thought. I'm having some trouble following all the various types but that might be because of the branches on top of branches making it harder to browse source right now.

Reply:
Updated `mentions` to `mentionElements`. Changed the `mentionsMap` to array form instead of Map and renamed to `mentionInternalsList` as the array been generated from the Map on parser.